vera-frontend/package 2.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Why do we have a second `package.json` file? Can we append these lines to the original file?

Reply:
I'm not sure why that happened, I deleted the file and its working fine

Reply:
Great, thanks!